__tests__/app.test.js

```diff
@@ -0,0 +1,90 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data");
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET:200 sends an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toEqual(expect.any(Array));
+        topics.forEach((topic) => {
```

Comment:
We should always check the length of an array before iterating over it. This will help our test fail faster if the issue is that the Array is empty, for example